test/adapter/polymorphic_test.rb

```diff
@@ -0,0 +1,72 @@
+require 'test_helper'
+
+module ActiveModel
+  class Serializer
+    module Adapter
```

Comment:
Thanks for the feedback @joaomdmoura. My first implementation was split out like you suggested, but I followed @bf4's suggestion to combine these, which has a benefit of being able to compare the payloads side by side. If you'd like, I'll revert the changes. Let me know the final verdict.